misc/python/materialize/cloudtest/util/sql.py

```diff
@@ -39,7 +39,7 @@ def sql_execute(
 
 def sql_execute_ddl(
     conn: Connection[Any],
-    query: str,
+    query: LiteralString,
```

Comment:
Are you allowed to pass a non-literal string?

Reply:
No. While you could provide some other types (e.g., `sql.SQL`), a raw string is not allowed (to prevent SQL injections).